app/models/petition.rb

```diff
@@ -53,12 +53,13 @@ def to_presentable_csv(options = {})
     end
   end
 
-  def to_affiliation_csv(institution, options = {})
+  def to_affiliation_csv(options = {})
```

Comment:
The call to this in `app/controllers/action_page_controller.rb` still uses the old method signature:

https://github.com/EFForg/action-center-platform/blob/3556b0c0bd36b40783dc3fdd392d27649f2c34ef/app/controllers/action_page_controller.rb#L60-L65